### ISSUE TITLE
Audio cues change sounds

### ISSUE THREAD

**Issue by BillDenton** [microsoft/vscode#165470]
Please add option to allow the audio cues to be changed. I'd like to use them for command success or failure for my builds. As the builds take a while I often switch away to another window, so I like to hear when they are done. The current sounds are distinctive enough so I'd like to use my own sound files.

**Comment by meganrogge** [comment]
reopened this one

/duplicate  #145204
